tailwind.config.js

```diff
@@ -3,46 +3,30 @@ import HeadlessTailwindPlugin from "@headlessui/tailwindcss";
 /** @type {import('tailwindcss').Config} */
 export default {
   content: ["./index.html", "./src/**/*.{html,js,jsx,ts,tsx}"],
-  mode: "jit",
-  purge: ["./public/**/*.html", "./src/**/*.{js,jsx,ts,tsx,vue}"],
+  content: ["./public/**/*.html", "./src/**/*.{js,jsx,ts,tsx,vue}"],
```

Comment:
Let's combine these separate `content` arrays into 1 array, the line below should cover all cases that these 2 arrays contain: 
`content: ["./index.html", "./public/**/*.html", "./src/**/*.{html,js,jsx,ts,tsx,vue}"]`